File: URI Programs/C/1043.c
#include <stdio.h>

int main(){	
	float a, b, c, result;
	scanf("%f %f %f", &a, &b, &c);
	if ((a + b > c) && (b + c > a) && (c + a > b)){
		result = a + b + c;
		printf("Perimetro = %.1f\n", result);
	} else {
		result = (a + b) * c / 2;
		printf("Area = %.1f\n", result);
	}
	return 0;
}

File: URI Programs/Python/1043.py
A, B, C = map(float, input().split())

if (A + B> C) and (B + C > A) and (C + A > B) :
  pe = A + B + C 
  print("Perimetro = "+str("%.1f" %pe))
else :
  a = (A + B)*C/2
  print("Area = "+str("%.1f" %a))